docs/tutorials/how-to-override-section-components.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "How to override or customize native section components"
+sidebar_label: "How to override or customize native section components"
+---
+
+
+# How to override or customize native section components
+This guide covers how to override native section components or pass additional props to them.
+
+Native sections are provided by default by VTEX's `@faststore/core` package. Sections are composed by different components that can be customized or overriden to add functionalities or to change looks.
+
+{Image or gif of the expected behavior after completing the guide}
+
+---
+
+## Before you start
+
+_Before you begin, you should already have:_
+- A functioning Evergreen store.
+
+_You should also consider that:_
+
+- This feature is experimental.
+- Some sections contain multiple instances of the same component. When overriting a component or passing additional props to it, consider that all instances will be affected by this.
+
+---
+
+## Step by step
+### Step 1 - Choose a native section to customize
+
+1. Choose the native section to be customized - [list of available native sections](./tbd-link)
+2. Navigate to `src/components/overrides` and create a new file named after the native section
+
+eg. Consider you chose the `ProductDetails` section to customize. Create a new file named `ProductDetails.tsx` under the `src/components/overrides` directory.
```

Comment:
I think it sits better as "Consider", since this is an arbitrary choice and just an example. "Considering" makes me think the reader had a set of options (it has, but it's not presented in the text) and that the possibility of them choosing other alternatives will be addressed later. It also reads as a suggestion, which is accurate in this scenario.